README.md

```diff
@@ -1,110 +1,148 @@
 
-The library exposes 3 classes:
+**Timezone-aware datetimes for Python that just work.**
 
-- `datetime_tz`, a datetime ensured to be timezone-aware.
-- `datetime_local`, a datetime ensured to be timezone-aware in the local timezone.
-- `datetime_utc`, a datetime ensured to be timezone-aware in the UTC+0 timezone.
+`heliclockter` is a timezone-aware datetime library that ensures your timestamps are always timezone-aware. It's statically type checkable and runtime enforceable.
 
-as well as various utilities to instantiate, mutate and serialize those classes.
+## Installation
 
-See our [announcement post][announcement] for a more background on why we wrote `heliclockter`.
-
-[pydantic]: https://github.com/pydantic/pydantic
-[announcement]: https://www.channable.com/tech/heliclockter-timezone-aware-datetimes-in-python
-
-Examples
--------
+```bash
+pip install heliclockter
+```
 
-Say you want to create a timestamp of the current time in the UTC+0 timezone.
+## Quick Start
 
 ```python
-from heliclockter import datetime_utc
+from heliclockter import datetime_utc, datetime_local, datetime_tz
 
-now = datetime_utc.now()
+# UTC datetime
+utc_now = datetime_utc.now()
 # datetime_utc(2022, 11, 4, 15, 28, 10, 478176, tzinfo=zoneinfo.ZoneInfo(key='UTC'))
-```
 
-Or imagine you want to create a timestamp 2 hours in the future from now:
+# Local timezone datetime  
+local_now = datetime_local.now()
 
-```python
-from heliclockter import datetime_utc
+# Any timezone datetime
+paris_tz = datetime_tz.now("Europe/Paris")
+
+# Create a timestamp 2 hours in the future
+future = datetime_utc.future(hours=2)
 
-two_hours_from_now = datetime_utc.future(hours=2)
-# datetime_utc(2022, 11, 4, 17, 28, 52, 478176, tzinfo=zoneinfo.ZoneInfo(key='UTC'))
+# Parse strings (naive timestamps assumed UTC)
+parsed = datetime_utc.strptime('2022-11-04T15:49:29', '%Y-%m-%dT%H:%M:%S')
 ```
 
-Features
---------
+## Why heliclockter?
 
-* Runtime enforcable timezone-aware datetimes
-* Utilities for instantiating, mutating and serializing timezone-aware datetimes
-* Statically type check-ble
-* Pydantic integration
-* Extensive test suite
-* No third party dependencies
+Python's standard `datetime` allows "naive" datetimes without timezone info, leading to bugs when:
+- Mixing naive and aware datetimes (causes runtime TypeErrors)
+- Deploying across different timezones
+- Forgetting to add `tzinfo` when creating datetimes
 
-Installation
-------------
+`heliclockter` enforces timezone-aware datetimes at the type level, catching these issues before production.
 
-To install `heliclockter`, simply: 
+## Key Features
 
-    $ pip install heliclockter
+- **Always timezone-aware** - No more naive datetime accidents
+- **Type safe** - Full typing support for better IDE experience  
+- **Zero dependencies** - Lightweight, uses only standard library
+- **Pydantic support** - Automatic integration when Pydantic is installed
+- **Python 3.9+** - Modern Python for modern applications
```

Comment:
We [recently](https://github.com/channable/heliclockter/pull/20) dropped support for 3.9, this should now be `3.10+`